aten/src/ATen/native/ForeachOpsKernels.cpp

```diff
@@ -21,4 +22,24 @@ void foreach_tensor_add_scalar_kernel_slow_(TensorList tensors, Scalar scalar) {
   }
 }
 
+std::vector<Tensor> foreach_tensor_add_list_kernel_slow(TensorList tensors1, TensorList tensors2) {
+  verify_list(tensors1, tensors2);
+
+  std::vector<Tensor> result;
+  for (int i = 0; i < tensors1.size(); i++) {
+    auto temp = tensors1[i].add(tensors2[i]);
+    result.emplace_back(temp);
```

Comment:
nit: this should be `result.emplace_back(std::move(temp))`, to avoid an extra refcount bump on Tensor. (example demonstrating that this code, without std::move, copies `temp`: https://godbolt.org/z/5reb1T)

Reply:
Or you can remove temp and just write ```result.push_back(tensors1[i].add(tensors2[i]))``` https://godbolt.org/z/M3qade